deploy/kubeturbo-operator/Makefile

```diff
@@ -217,34 +222,44 @@ create_certified_operator_bundle_directory:
 	mkdir -p $(OPERATOR_BUNDLE_DIR)/metadata/
 	cp $(OPERATOR_BUNDLE_TEMPLATE_DIR)/template/annotations.yaml $(OPERATOR_BUNDLE_DIR)/metadata/annotations.yaml
 	@echo "$(OPERATOR_CERTIFIED)-operator bundle directory created successfully."
+
 ## update image digest key
 update_image_digest_in_operator_bundle:
 	@echo "Updating image digest in $(OPERATOR_CERTIFIED)-clusterserviceversion..."
 	digest=$$(docker inspect --format='{{index .RepoDigests 0}}' $(REGISTRY)/$(OPERATOR_NAME)-operator:$(OPERATOR_RELEASE_VERSION) | awk -F@ '{print $$2}'); \
-	$(YQ) eval -i '.spec.install.spec.deployments[0].spec.template.spec.containers[0].image |= sub("sha256:.*", "'$$digest'") | .spec.relatedImages[0].image |= sub("sha256:.*", "'$$digest'")' \
-		$(OPERATOR_BUNDLE_DIR)/manifest/$(OPERATOR_CERTIFIED).clusterserviceversion.yaml
-	@echo "$(OPERATOR_CERTIFIED)-clusterserviceversion image digest updated."
+	if [ -z "$$digest" ]; then \
+		echo "Error: Image digest is empty, cannot proceed with $(OPERATOR_CERTIFIED)-operator bundle release."; \
+		exit 1; \
+	else \
+	    $(YQ) eval -i '.spec.install.spec.deployments[0].spec.template.spec.containers[0].image |= sub("sha256:.*", "'$$digest'") | .spec.relatedImages[0].image |= sub("sha256:.*", "'$$digest'")' \
+		    $(OPERATOR_BUNDLE_DIR)/manifest/$(OPERATOR_CERTIFIED).clusterserviceversion.yaml; \
+	    echo "$(OPERATOR_CERTIFIED)-clusterserviceversion image digest updated."; \
+    fi
```

Comment:
Nit: Indentation

Reply:
Not sure why it's showing this way, but it's correctly aligned when we pull and open in IDE. I verified on vscode and IntelliJ